classes/WP_CLI/System_Command.php

```diff
@@ -180,6 +184,10 @@ public function source( array $args, array $assoc_args ) {
 
 		ksort( $rows );
 
+		if ( ! $skipnote ) {
+			\WP_CLI::line( PHP_EOL . 'Please note there can only be one unique registered instance of Action Scheduler per ' . PHP_EOL . 'version number, so this list may not include all the currently present copies of ' . PHP_EOL . 'Action Scheduler.' . PHP_EOL );
```

Comment:
Wondering now if it is or is not a good practice to manually "word-wrap" like this 🤔

Reply:
Following this train-of-thought (I was curious if WP CLI might either have some notes on best practices, or else have a helper that manages this for us) I noticed that perhaps we should [now prefer `::log()` instead of `::line()`](https://make.wordpress.org/cli/handbook/references/internal-api/wp-cli-line/#notes):

> Message is written to STDOUT. WP_CLI::log() is typically recommended;
WP_CLI::line() is included for historical compat.

Thinking about it, using `::log()` may be more elegant because it pretty much [removes the need for us to implement `--skip-notice`](https://make.wordpress.org/cli/handbook/references/internal-api/wp-cli-log/#notes) (which otherwise was a nice touch)

> Message is written to STDOUT, or discarded when --quiet flag is supplied.